web/src/js/__tests__/components/FlowView/MessagesSpec.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import TestUtils from 'react-dom/test-utils'
+import { Request, Response, ErrorView } from '../../../components/FlowView/Messages'
+import { Provider } from 'react-redux'
+import { TFlow, TStore } from '../../ducks/tutils'
+import { updateEdit } from '../../../ducks/ui/flow'
+import { parseUrl } from '../../../flow/utils'
+import ContentView from '../../../components/ContentView'
+import ContentViewOptions from '../../../components/ContentView/ContentViewOptions'
+import Headers from '../../../components/FlowView/Headers'
+import ValueEditor from '../../../components/ValueEditor/ValueEditor'
+import mockXMLHttpRequest from 'mock-xmlhttprequest'
+
+global.XMLHttpRequest = mockXMLHttpRequest
+global.fetch = jest.fn()
+
+let tflow = new TFlow(),
+    store = TStore()
+store.getState().ui.flow.modifiedFlow = false
+
+mockXMLHttpRequest.onSend = xhr => {
```

Comment:
The `ContentView` component here will invoke `updateContent` in `ContentLoad`, that function use XMLHttpRequest. So I mocked XMLHttpRequest here to feed `updateContent`.
But when I reconsider this just now, I found that we could actually mock the `ContentView` to bypass the error. I'll update the new version.